README.md

```diff
@@ -88,6 +88,10 @@ require('copilot').setup({
       refresh = "gr",
       open = "<M-CR>"
     },
+    position = {
+      split = "horizontal",
+      size = 0.4
+    }
```

Comment:
`size` inside `position` is weird. Maybe rename this to:

```lua
layout = {
  position = "top", -- or bottom or left or right
  size = 0.4,
}
```

Reply:
Thanks,, that sounds much better! Since it's less code to implement and adds more customization options, what do you think about  just taking any valid split position option like`belowright` `belowleft` ... `topright` etc?

Reply:
Not all of those are built-in position tho tho. e.g. there's no `topright`, only `topleft`.

Also with those, you'll also need to know if it's ver/hor. e.g. `topleft` goes to top for hor split, but goes to left for ver split.

Taking `top/bottom/left/right` and translating it internally is better. Check this: https://github.com/MunifTanjim/nui.nvim/blob/257da38029d3859ed111804f9d4e95b0fa993a31/lua/nui/split/init.lua#L17-L20

Reply:
looks good... there are some inconsistent naming in different places (e.g. layout vs. position, split vs. position, size vs ratio).

Reply:
I went through and searched for the old naming scheme to replace it, it should be layout { position, ratio } everywhere, but I must have missed something. I'll go through and fix it now